### PR TITLE
Add increment util method to increment counter metrics in statsd

### PR DESCRIPTION
**Pull request by pchheda-lyft**
Adding this util method will allow us to support counter metrics publishing to statsd.

### CODE REVIEW

**Review by achantavy — DISMISSED** [review]
Thanks for doing this!

File: cartography/util.py [hunk 1]
```diff
@@ -68,6 +68,19 @@ def timed(*args, **kwargs):
     return timed
 
 
+def increment(stat, count=1, rate=1):
```
Comment (achantavy) [inline cartography/util.py:71]: Can you add typehints here and maybe rename this to something clearer - what is actually being incremented here?

Comment (pchheda-lyft) [reply cartography/util.py:71]: Sure. Let me do that!

Comment (pchheda-lyft) [reply cartography/util.py]: Refactored

**Review by achantavy — APPROVED** [review]

**Comment by gkleiman** [comment]
Have we considered wrapping this in a class (e.g., `StatsClient`) that takes a scope/prefix?

You could have something like:

```
class ScopedStatsClient:
    def __init__(self, parent, scope = None):
         self.prefix = scope
         self.parent = parent

    def scopedClient(self, scope):
         return ScopedStatsClient(self, scope)

    def incr(self, name, count=1, rate=1):
         self.parent.incr(f'{self.prefix}.name, count, rate)
```

**Comment by FAYiEKcbD0XFqF2QK2E4viAHg8rMm2VbjYKdjTg** [comment]
I like Gaston's suggestion. @achantavy any objections?

**Comment by achantavy** [comment]
I don't have a strong opinion either way. The scenario is to have counter metrics, and having it scoped or not scoped isn't really a blocker, and I think it wouldn't be _that_ bad to refactor later on for scoping. Up to you, @pchheda-lyft

**Pull request closed without merging**